source/seedsource/tasks/cleanup_tifs.py

```diff
@@ -0,0 +1,20 @@
+from seedsource_project import settings
+import os
+import os.path
+import time
+import re
+from celery.task import task
+
+@task
+def cleanup_temp_tif_files(age=7200):
+    tempDir = settings.NC_TEMPORARY_FILE_LOCATION
```

Comment:
Please use underscored words instead of camel case for local variables: `temp_dir`.